tests/test_utils.py

```diff
@@ -0,0 +1,68 @@
+import torch
+import numpy as np
+
+from cata2data import collate_fn_irregular_cutouts, collate_fn_regular_cutouts
+
+def test_collate_fn_irregular_cutouts_with_arraylike_y():
+    batch = [
+        (np.random.rand(3, 4), np.array([1, 2, 3])),
+        (np.random.rand(3, 4), np.array([4, 5, 6])),
+    ]
+    x, y = collate_fn_irregular_cutouts(batch)
+
+    assert isinstance(x, list), "Expected x to be a list."
+    assert isinstance(y, torch.Tensor), "Expected y to be a tensor."
+    assert len(x) == 2, "Expected x to have length 2."
+    assert y.shape == (6,), "Expected y to be concatenated correctly."
+
+def test_collate_fn_irregular_cutouts_with_scalar_y():
+    batch = [
+        (np.random.rand(3, 4), 1),
+        (np.random.rand(3, 4), 0),
+    ]
+    x, y = collate_fn_irregular_cutouts(batch)
```

Comment:
This should probably actually test irregular cutouts. I.e. different input data shapes:
```suggestion
def test_collate_fn_irregular_cutouts_with_scalar_y():
    batch = [
        (np.random.rand(1, 2), 1),
        (np.random.rand(3, 4), 0),
    ]
    x, y = collate_fn_irregular_cutouts(batch)
```